tools/cldr/cldr-to-icu/build-icu-data.xml

```diff
@@ -61,10 +61,10 @@
         <property name="specialsDir" value="${basedir}/../../../icu4c/source/data/xml"/>
 
         <!-- Default value for ICU version (icuver.txt). Update this for each release. -->
-        <property name="icuVersion" value="75.0.1.0"/>
+        <property name="icuVersion" value="75.1.0.0"/>
 
         <!-- Default value for ICU data version (icuver.txt). Update this for each release. -->
-        <property name="icuDataVersion" value="75.0.1.0"/>
+        <property name="icuDataVersion" value="75.1.0.0"/>
```

Comment:
@mihnita @dra8an @pedberg-icu @echeran @yumaoka where do these numbers go?
Do they go into icuver.txt?
Shouldn't these be updated as part of the CLDR-to-ICU integration?
Do we need to rerun the CLDR-to-ICU integration after this PR is merged?

Reply:
I am not familiar with that code.

But I did a cldr-to-icu text file generation with these properties set to 75.0.1.0 and 75.1.0.0 (this PR, basically)

The only difference is in `icu4c/source/data/misc/icuver.txt` and a few time stamps (not versions) in some maven artifacts.

Since icuver.txt seems to be already updated (commit 8ba19195f94, 2024-03-22), there is no need to rerun the CLDR-to-ICU integration.